File: SDLC/SDLC1/W4/Eindopdr/MastermindCore6/MastermindCore6/MastermindCore6/Model/Uitslag.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MastermindCore6.Model
{
    public class Uitslag
    {
        public string Poging { get; private set; }
        public int Posities { get; private set; }
        public int Aantal { get; private set; }
        private int beurt;

        public Uitslag(int beurt, string poging, int pos, int aantal)
        {
            this.beurt = beurt + 1;
            Poging = poging;
            Posities = pos;
            Aantal = aantal;
        }
        public override string ToString()
        {
            return $"{beurt} \t {Poging} \t\t {Posities} \t {Aantal}";
        }

    }
}

File: SDLC/SDLC1/W5/mastermindv81/mastermindv8/mastermindv8/app/Models/Uitslag.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Uitslag extends Model
{
    public $Poging;
    public $Posities;
    public $Aantal;
    private $beurt;

    public function Uitslag($beurt, $poging, $pos, $aantal)
    {
        $this->beurt = $beurt + 1;
        $this->Poging = $poging;
        $this->Posities = $pos;
        $this->Aantal = $aantal;
    }
    public function ToString()
    {
        return "$this->beurt \t $this->Poging \t\t $this->Posities \t $this->Aantal";
    }
}
